pvt/benchmarks/joinExit.ts

```diff
@@ -14,7 +14,7 @@ import {
 import { deployedAt } from '@balancer-labs/v2-helpers/src/contract';
 
 // setup environment
-const BPTAmount = bn(5e18);
+const BPTAmount = bn(1e18);
```

Comment:
This seems very minor but just wondering what's the reason for this change?

Reply:
Otherwise it runs out of balance when you run it with 20 tokens. I didn't try other values; maybe 2 or 3 would work